pom.xml

```diff
@@ -242,13 +244,29 @@
       <artifactId>google-metadata-plugin</artifactId>
       <version>0.2</version>
     </dependency>
-
+    <!--                   /////////////////////////////   -->
+    <dependency>
+      <groupId>com.google.http-client</groupId>
```

Comment:
Could you give more context for these changes?

Reply:
Because plugin doesn't work without this dependency ? :)))))

Reply:
Update to Java 1.8 and you will see that plugin doesn't work with newer Dependence for 1.8JDK